pull/github.go

```diff
@@ -516,77 +516,62 @@ func (ghc *GitHubContext) RepositoryCollaborators() ([]*Collaborator, error) {
 		// should only be used when assigning user reviewers, in which case
 		// almost all of the calls would have been made anyway.
 
-		var q struct {
-			Repository struct {
-				DirectCollaborators struct {
-					PageInfo v4PageInfo
-					Edges    []struct {
-						Permission string
-					}
-					Nodes []v4Actor
-				} `graphql:"direct: collaborators(affiliation: DIRECT, first: 50, after: $directCursor)"`
-				AllCollaborators struct {
-					PageInfo v4PageInfo
-					Edges    []struct {
-						Permission string
-					}
-					Nodes []v4Actor
-				} `graphql:"all: collaborators(affiliation: ALL, first: 50, after: $allCursor)"`
-			} `graphql:"repository(owner: $owner, name: $name)"`
-		}
-		qvars := map[string]interface{}{
-			"owner":        githubv4.String(ghc.owner),
-			"name":         githubv4.String(ghc.repo),
-			"directCursor": (*githubv4.String)(nil),
-			"allCursor":    (*githubv4.String)(nil),
+		permissionToGitHub := map[Permission]string{
```

Comment:
Instead of having this map here, let's define a `GitHubString()` method on the `Permission` type that returns the strings as used in the GitHub API.